docker.js

```diff
@@ -0,0 +1,110 @@
+var path = require('path')
+var ncp = require('ncp')
+var mkdirp = require('mkdirp')
+var fs = require('fs')
+var cp = require('child_process')
+var fetch = require('node-fetch')
+//run lambda process within docker.
+
+function copy (src, dest, cb) {
```

Comment:
nitpick: we try to consistently use async/await in all our code and avoid callbacks. can discuss in the code session / call tonight